demo/kitchen_sink_random/run.py

```diff
@@ -0,0 +1,102 @@
+)
+
+
+demo = gr.Interface(
+    lambda x: x,
+    inputs=[
+        gr.Textbox(value=lambda: datetime.now(), label="Current Time"),
+        gr.Number(value=lambda: random.random(), label="Ranom Percentage"),
+        gr.Slider(minimum=-1, maximum=1, randomize=True, label="Slider with randomize"),
+        gr.Slider(
+            minimum=0,
+            maximum=1,
+            value=lambda: random.random(),
+            label="Slider with value func",
+        ),
+        gr.Checkbox(value=lambda: random.random() > 0.5, label="Random Checkbox"),
+        gr.CheckboxGroup(
+            choices=["a", "b", "c", "d"],
+            value=lambda: random.choice(["a", "b", "c", "d"]),
+            label="Random CheckboxGroup",
+        ),
+        gr.Radio(
+            choices=list(string.ascii_lowercase),
+            value=lambda: random.choice(string.ascii_lowercase),
+        ),
+        gr.Dropdown(
+            choices=["a", "b", "c", "d", "e"],
+            value=lambda: random.choice(["a", "b", "c"]),
+        ),
+        gr.Image(
+            value=lambda: random.choice(
+                [os.path.join(img_dir, img) for img in os.listdir(img_dir)]
+            )
+        ),
+        gr.Video(value=lambda: os.path.join(file_dir, "world.mp4")),
+        gr.Audio(value=lambda: os.path.join(file_dir, "cantina.wav")),
+        gr.File(
+            value=lambda: random.choice(
+                [os.path.join(file_dir, img) for img in os.listdir(file_dir)]
+            )
+        ),
+        gr.Dataframe(
+            value=lambda: pd.DataFrame(
+                {"random_number_rows": range(random.randint(0, 10))}
+            )
+        ),
+        gr.Timeseries(value=lambda: os.path.join(file_dir, "time.csv")),
+        gr.Variable(value=lambda: random.choice(string.ascii_lowercase)),
+        gr.Button(value=lambda: random.choice(["Run", "Go", "predict"])),
+        gr.ColorPicker(value=lambda: random.choice(["#000000", "#ff0000", "#0000FF"])),
+        gr.Label(value=lambda: random.choice(["Pedestrian", "Car", "Cyclist"])),
+        gr.HighlightedText(
+            value=lambda: random.choice(
+                [
+                    {"text": highlighted_text, "entities": highlighted_text_output_1},
+                    {"text": highlighted_text, "entities": highlighted_text_output_2},
+                ]
+            ),
+        ),
+        gr.JSON(value=lambda: random.choice([{"a": 1}, {"b": 2}])),
+        gr.HTML(
+            value=lambda: random.choice(
+                [
+                    '<p style="color:red;">I am red</p>',
+                    '<p style="color:blue;">I am blue</p>',
+                ]
+            )
+        ),
+        gr.Gallery(
+            value=lambda: [os.path.join(img_dir, img) for img in os.listdir(img_dir)]
+        ),
+        gr.Chatbot(
+            value=lambda: random.choice([[("hello", "hi!")], [("bye", "goodbye!")]])
+        ),
+        gr.Model3D(
+            value=lambda: random.choice(
```

Comment:
For some reason Model3D is not being randomized on the front-end even though it is on the back-end.

My guess is that the model3d component is not being cleared?

```
To create a public link, set `share=True` in `launch()`.
/Users/freddy/sources/gradio/demo/kitchen_sink_random/../model3D/files/Bunny.obj
/Users/freddy/sources/gradio/demo/kitchen_sink_random/../model3D/files/Bunny.obj
/Users/freddy/sources/gradio/demo/kitchen_sink_random/../model3D/files/face.obj
/Users/freddy/sources/gradio/demo/kitchen_sink_random/../model3D/files/Bunny.obj
/Users/freddy/sources/gradio/demo/kitchen_sink_random/../model3D/files/face.obj
/Users/freddy/sources/gradio/demo/kitchen_sink_random/../model3D/files/Fox.gltf
```

Don't think that should hold up this PR though.